crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -195,9 +195,15 @@ def prepare_nodes_for_crowbar_upgrade
     # To all nodes, add a new role which prepares them for the upgrade
     nodes_to_upgrade = []
     not_ready_for_upgrade = []
+    admin_with_dns_server_role = false
```

Comment:
As we are trying to look for incorrect case, it might be better to actually have `admin_without_dns_server_role` (but I do not care that much)